test/localhost.crt

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Oh I was thinking that everyone would just use these that are checked in. The script is just there in case anyone wants to run it again in the future (e.g. tweak the various parameters here and there) or create new certs for new tests.